cantor-h2/src/main/java/com/salesforce/cantor/h2/EventsOnH2.java

```diff
@@ -95,6 +95,23 @@ protected String getCreateChunkTableSql(final String chunkTableName,
         return sqlBuilder.toString();
     }
 
+    @Override
+    protected String getRegexPattern(final String originalPattern) {
+        String finalPattern = originalPattern.replaceAll("\\*", "\\.\\*");
+
+        // To match a prefix, regex pattern starts with "^"
+        if (!originalPattern.startsWith("*")) {
+            finalPattern = "^" + finalPattern;
+        }
+
+        // To match a postfix, regex pattern ends with "$"
+        if (!originalPattern.endsWith("*")) {
+            finalPattern = finalPattern + "$";
+        }
+
+        return finalPattern;
+    }
```

Comment:
H2 does not support LIKE operators: http://www.h2database.com/html/functions.html